test/fake_http_server.cc

```diff
@@ -18,7 +18,7 @@ FakeServer::~FakeServer() {
 
 std::string FakeServer::GetUrl() {
   network::uri_builder builder;
-  builder.scheme("http").host(server_.address()).port(server_.port()).path("/");
```

Comment:
Can you please provide the rationale for this change?

Reply:
KubernetesEndpointHost does not want a trailing slash.

Reply:
Sigh...

Reply:
Acknowledged.